server/app.js

```diff
@@ -27,9 +29,9 @@ app.use('/users', usersRouter);
 app.use('/todo', todoRouter);
 
 // catch 404 and forward to error handler
-app.use(function (req, res, next) {
-    next(createError(404));
-});
+// app.use(function (req, res, next) {
```

Comment:
nitpick - can this commented code be removed

Reply:
it shall be done